hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -1232,14 +1232,14 @@
       name: chip_sw_alert_handler_entropy
       desc: '''Verify the alert handler entropy input to ensure pseudo-random ping timer.
 
-            - Force `alert_handler_ping_timer` input signal `wait_cyc_mask_i` to `4'bff` to shorten
-              the simulation time.
+            - Force `alert_handler_ping_timer` input signal `wait_cyc_mask_i` to `12'h7ff` to
```

Comment:
```suggestion
            - Force `alert_handler_ping_timer` input signal `wait_cyc_mask_i` to `8'hff` to
```

Reply:
Thanks Matute, but I am forcing value `7ff` to this port. Should I just use `h7ff` instead?